pkgs/applications/blockchains/bitcoin/default.nix

```diff
@@ -121,6 +121,7 @@ stdenv.mkDerivation (finalAttrs: {
       ln -s $src ./bitcoin-${finalAttrs.version}.tar.gz
       gpg --no-autostart --batch --verify --status-fd 1 SHA256SUMS.asc SHA256SUMS > verify.log
       ${verifyBuilderKeys}
+      echo Checking ${checksums} for bitcoin-${finalAttrs.version}.tar.gz
```

Comment:
Maybe surround the echo statement with double quotes and add a `...` at the end to match the style of the rest of the echo statements.